handler/handler.go

```diff
@@ -83,6 +83,9 @@ func GetTaskInfo(scheduler types.Scheduler) http.HandlerFunc {
 		id := vars["taskId"]
 		logrus.WithField("task_id", id).Debug("Fetching task")
 		task, _ := database.ReadTask(id)
```

Comment:
Defaulted ReadTask to masking, and added a ReadUnmaskedTask to explicitly say you want the raw data.